packages/vaadin-rich-text-editor/gulpfile.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-env node */
+'use strict';
+
+const gulp = require('gulp');
+const iconfont = require('gulp-iconfont');
+const exec = require('child_process').exec;
+const fs = require('fs');
+
+gulp.task('icons', function (done) {
+  var glyphs;
+  const fontName = 'vaadin-rte-icons';
+  const fileName = 'vaadin-rich-text-editor-icons';
+
+  gulp
+    .src('icons/*.svg')
+    .pipe(
+      iconfont({
+        fontName: fileName,
+        formats: ['woff'],
+        fontHeight: 1000,
+        ascent: 850,
+        descent: 150,
+        fixedWidth: true,
+        normalize: true,
+        timestamp: 1 // Truthy!
+      })
+    )
+    .on('glyphs', function (glyphData) {
+      // Store for later use
+      glyphs = glyphData;
+    })
+    .pipe(gulp.dest('.'))
+    .on('finish', function () {
+      // Generate base64 version of the font
+      exec(`base64 ${fileName}.woff`, function (err, stdout) {
```

Comment:
Why it uses a different way to get the font in the base64 format? 

Is there any specific reason behind it? 

Probably, it can be aligned with the other methods:

```
const materialIconsWoff = fs.readFileSync('material-icons.woff');

materialIconsWoff.toString('base64')
```

Reply:
Good catch, changed.